dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryGlobalOpenTelemetryTest.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat;
+
+import static com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions.SERVER_ONLY;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.globalopentelemetry.TelemetryGlobalOpenTelemetryServlet;
+
+@RunWith(FATRunner.class)
+public class TelemetryGlobalOpenTelemetryTest extends FATServletClient {
+
+    public static final String SERVER_NAME = "Telemetry10";
+    public static final String APP_NAME = "TelemetryApp";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        WebArchive app = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                        .addClasses(TelemetryGlobalOpenTelemetryServlet.class)
+                        .addAsResource(new StringAsset("otel.sdk.disabled=false"),
+                                       "META-INF/microprofile-config.properties");
+        ShrinkHelper.exportAppToServer(server, app, SERVER_ONLY);
+        server.startServer();
+    }
+
+    @Test
+    public void testSetGlobalOpenTelemetry() throws Exception {
+        runTest(server, APP_NAME + "/GlobalOpenTelemetryServlet", "testSetGlobalOpenTelemetry");
+    }
+
+    //A warning should only be shown the first time the user attempts to get the GlobalOpenTelemetry instance
+    @Test
+    public void testGetGlobalOpenTelemetry() throws Exception {
+        server.setMarkToEndOfLog();
+        runTest(server, APP_NAME + "/GlobalOpenTelemetryServlet", "testGetGlobalOpenTelemetry");
+        assertFalse(server.waitForStringInLogUsingMark("CWMOT5000W: Cannot get GlobalOpenTelemetry.").isEmpty());
```

Comment:
`waitForStringInLogUsingMark` returns `null` if the string is not found. This line will throw a `NullPointerException` if the warning isn't emitted.